core/src/test/kotlin/generator/TestRunner.kt

```diff
@@ -0,0 +1,93 @@
+package generator
+
+import org.jetbrains.dokka.*
+import org.jetbrains.dokka.utilities.DokkaConsoleLogger
+import org.jetbrains.dokka.utilities.DokkaLogger
+import java.io.IOException
+import java.nio.charset.Charset
+import java.nio.file.*
+import java.nio.file.attribute.BasicFileAttributes
+
+
+object TestRunner {
+
+    fun generateTestDokka(
+        name: String,
+        query: String,
+        checks: Map<DokkaTestGenerator.Companion.Stage, (DokkaTestGenerator) -> Boolean>,
+        targets: List<String> = listOf("jvm"),
+        logger: DokkaLogger = DokkaConsoleLogger
+    ) {
+        val root = Paths.get(".").resolve("build").resolve("tests").resolve(name)
+        root.deleteRec()
```

Comment:
Do not hardcode paths, we should probably use temporary directories for this

Reply:
Done.